Sources/SmileID/Classes/Networking/ServiceRunnable.swift

```diff
@@ -193,20 +221,33 @@ extension ServiceRunnable {
     private func createRestRequest<T: Encodable>(
         path: PathType,
         method: RestMethod,
-        headers: [HTTPHeader]? = nil,
+        headers: [HTTPHeader],
         queryParameters: [HTTPQueryParameters]? = nil,
         body: T
     ) async throws -> RestRequest {
         let path = String(describing: path)
-        guard let url = baseURL?.appendingPathComponent(path) else {
+        guard var url = baseURL?.appendingPathComponent(path) else {
```

Comment:
There is an Xcode warning here `Variable 'url' was never mutated; consider changing to 'let' constant`.